drivers/counter/counter_nrfx_rtc.c

```diff
@@ -22,10 +23,16 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL);
 #define TOP_CH 0
 #define COUNTER_TOP_INT NRFX_RTC_INT_COMPARE0
 
+#define IS_PPI_WRAP() \
+	(IS_ENABLED(CONFIG_COUNTER_RTC0_PPI_WRAP) || \
+	 IS_ENABLED(CONFIG_COUNTER_RTC1_PPI_WRAP) || \
+	 IS_ENABLED(CONFIG_COUNTER_RTC2_PPI_WRAP))
+
 struct counter_nrfx_data {
 	counter_top_callback_t top_cb;
 	void *top_user_data;
 	u32_t top;
+	nrf_ppi_channel_t ppi_ch;
```

Comment:
This could be wrapped in `#if IS_PPI_WRAP()`.

Reply:
done